### ISSUE TITLE
modify_order -> assert not working

### ISSUE THREAD

**Issue by pfpietro** [ftexchange/ftx#88]
@authentication_required
    def modify_order(
        self, existing_order_id: Optional[str] = None,
        existing_client_order_id: Optional[str] = None, price: Optional[float] = None,
        size: Optional[float] = None, client_order_id: Optional[str] = None,
    ) -> dict:
        assert (existing_order_id is None) ^ (existing_client_order_id is None), \
            'Must supply exactly one ID for the order to modify'
        **assert (price is None) or (size is None), 'Must modify price or size of order'**
        path = f'orders/{existing_order_id}/modify' if existing_order_id is not None else \
            f'orders/by_client_id/{existing_client_order_id}/modify'
        return self._post(path, {
            **({'size': size} if size is not None else {}),
            **({'price': price} if price is not None else {}),
            ** ({'clientId': client_order_id} if client_order_id is not None else {}),
        })

So if you submit a modify_order passing a new price and a new size order there is going to be a exception:
assert (price is not None) or (size is not None), '**Must modify price or size of order'**

Is this supposed to work like this ? 
I believe we need to change it to NOT NONE

**Comment by BCusack** [comment]
Yes, It's one or the other not both from FTX Docs https://docs.ftx.com/#modify-order 
This could be handles cleaner, but can work around it.
Just cancels the existing order and replaces -> so this operation could be done instead.

![image](https://user-images.githubusercontent.com/10039059/147798379-30623b72-59b6-4ab8-a785-875d63e6e8ee.png)
